src/components/DocumentList/DocumentListScreenContainer.test.tsx

```diff
@@ -71,46 +82,4 @@ describe("DocumentListScreenContainer", () => {
       }
     );
   });
-
-  it("should navigate to QrScannerScreen when db is empty and add is pressed", () => {
```

Comment:
Thinking of leaving these in since it test the container's navigation logic

Reply:
Either that or you can test that the navigation bar exist?

Reply:
Yea, good point, I've added it back in since `add` in `EmptyDocumentList` is agnostic to the target navigation route